content/md/productivity/manage-your-rules.md

```diff
@@ -89,6 +103,9 @@ Depending on the action you choose, the template changes. The table below lists
 | Set attribute value  | Replace the attribute value of your product/product models selection  |
 | Copy attribute value  | Copy the value from an attribute to another one  |
 | Remove categories  | Remove the categories of your product/product model selection  |
+| Set attribute values  | Set options to a multi-select attribute, set value to a number attribute, set value to a boolean attribute. It will remove the former ones |
```

Comment:
```suggestion
| Set attribute values  | Set options to a multi select, number or boolean attribute. It will remove the former ones |
```
J'ai pas compris la dernière phrase.